packages/web-components/src/components/leadspace/__stories__/leadspace.stories.ts

```diff
@@ -39,24 +39,24 @@ const navigationWithTagGroup = html`
 `;
 
 const navigationWithBreadcrumbs = html`
-  <bx-breadcrumb slot="navigation">
-    <bx-breadcrumb-item>
-      <bx-breadcrumb-link href="/#">Breadcrumb 1</bx-breadcrumb-link>
-    </bx-breadcrumb-item>
-    <bx-breadcrumb-item>
-      <bx-breadcrumb-link href="/#">Breadcrumb 2</bx-breadcrumb-link>
-    </bx-breadcrumb-item>
-    <bx-breadcrumb-item>
-      <bx-breadcrumb-link href="/#" aria-current="page">Breadcrumb 3</bx-breadcrumb-link>
-    </bx-breadcrumb-item>
-  </bx-breadcrumb>
+  <dds-breadcrumb slot="navigation">
+    <dds-breadcrumb-item>
+      <dds-breadcrumb-link href="/#">Breadcrumb 1</dds-breadcrumb-link>
+    </dds-breadcrumb-item>
+    <dds-breadcrumb-item>
+      <dds-breadcrumb-link href="/#">Breadcrumb 2</dds-breadcrumb-link>
+    </dds-breadcrumb-item>
+    <dds-breadcrumb-item>
+      <dds-breadcrumb-link href="/#" aria-current="page">Breadcrumb 3</dds-breadcrumb-link>
+    </dds-breadcrumb-item>
+  </dds-breadcrumb>
 `;
 
 export const TallWithNoImage = ({ parameters }) => {
   const { alt, defaultSrc, title, copy, buttons, navElements } = parameters?.props?.LeadSpace ?? {};
   return html`
     <dds-leadspace size="${LEADSPACE_SIZE.NONE}" alt="${ifNonNull(alt)}" default-src="${ifNonNull(defaultSrc)}">
-      ${navElements === navigationOptions[0] ? console.log('hello') : ``}
```

Comment:
👋

Reply:
haha yeah this line is removed and fixed below 👍